backend/resources/views/security/index.blade.php

```diff
@@ -6,6 +6,7 @@
 @endsection
 @section('content')
 <h1 class="text-center my-8 text-3xl ">セキュリティ</h1>
-
+<h2>登録日: {{$registerDate}}</h2>
```

Comment:
上記の変更に合わせまして……
```suggestion
<h2>登録日: {{$user->created_at}}</h2>
```
でお願いします！